common/hooks/post-install/14-fix-permissions.sh

```diff
@@ -0,0 +1,19 @@
+# This hook fixes permissions in common places
+
+change_file_perms() {
+	dir="$PKGDESTDIR$1"
+	perms="$2"
+	if [ -d "$dir" ]; then
+		find "$dir" -type f | while read -r file; do
```

Comment:
find "$dir" -type f ! -perm -0644

Reply:
Or even better:
```find /usr/include/ -type f ! -perm -"$perms" -exec chown "$perms" '{}' +```

Reply:
`! -perm -0600` would match a file with permissions 700, but not a file with permissions 670. I modified the function to accept a permission mask as its second argument.

> Or even better:
> find /usr/include/ -type f ! -perm -"$perms" -exec chmod -v "$perms" '{}' +

Is this better? It's shorter, but also we lose the pretty warning message.

Reply:
Shorter and faster and `chmod -v` still gives you messages.

Reply:
Thank you, fixed.